<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemAppendWindow</class>
 <widget class="QWidget" name="ItemAppendWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>1279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="big_widget" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>461</width>
     <height>481</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="photo_special">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>160</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>160</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget{
	
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Change Photo</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="photo_type_selected">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Next release data:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTimeEdit" name="time"/>
        </item>
        <item>
         <widget class="QDateEdit" name="date"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="radioButton_layout">
        <item>
         <widget class="QRadioButton" name="radioButton_anime">
          <property name="text">
           <string>Anime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_manga">
          <property name="text">
           <string>Movie</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="episode_data_layout">
      <item>
       <layout class="QVBoxLayout" name="text_info_ep">
        <item>
         <widget class="QLabel" name="label_ep_t">
          <property name="text">
           <string>Total number of episodes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_ep">
          <property name="text">
           <string>Last episode</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="number_data_ep" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QSpinBox" name="total_ep"/>
         </item>
         <item>
          <widget class="QSpinBox" name="last_ep"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="name_text"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cancel_button">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_button">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
